deepchecks/vision/checks/model_evaluation/class_performance.py

```diff
@@ -273,19 +277,18 @@ def condition(check_result: pd.DataFrame) -> ConditionResult:
                 max_value = max_row['Value']
 
                 relative_difference = abs((min_value - max_value) / max_value)
-
-                if relative_difference >= threshold:
-                    details = (
-                        f'Relative ratio difference between highest and lowest in {dataset} dataset '
-                        f'classes is {format_percent(relative_difference)}, using {score} metric. '
-                        f'Lowest class - {min_class_name}: {format_number(min_value)}; '
-                        f'Highest class - {max_class_name}: {format_number(max_value)}'
-                    )
-                    datasets_details.append(details)
-            if datasets_details:
-                return ConditionResult(ConditionCategory.FAIL, details='\n'.join(datasets_details))
-            else:
-                return ConditionResult(ConditionCategory.PASS)
+                condition_passed &= relative_difference < threshold
+
+                details = (
+                    f'Relative ratio difference between highest and lowest in {dataset} dataset '
```

Comment:
Why we need to say what is the dataset, isnt it written elsewhere already?

Reply:
no, this is a train-test check so we are running on both